.env.sample

```diff
@@ -21,6 +21,12 @@ NAME=cs310
 ## created inside it.
 ORG=CS310-2017Jan
 
+## CLASSLIST API INTEGRATION
+## https://{host}:{port}/classlist1.0/CPSC/{courseNum}/{YEAR}{SEMESTER W or S}/{SectionNumbers}
```

Comment:
I think a more standard endpoint might just be `https://{host}:{port}/classlist/v1/CPSC/{courseNum}/{YEAR}{SEMESTER W or S}/{SectionNumbers}` but that's just personal preference.

Reply:
Agreed and fixed.